.travis.yml

```diff
@@ -13,15 +32,14 @@ julia:
   - 1.0
   - nightly
 env:
-  - JULIA_PROJECT="@."
+  - JULIA_TZ_VERSION="2016j"  # Ideally only set for the "test" stage
 notifications:
   email: false
-env:
-  - JULIA_TZ_VERSION="2016j"
 script:
   - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
   # Avoid setting JULIA_TZ_VERSION if the deps/build.jl file has been modified
   - git fetch origin +:refs/remotes/origin/HEAD; if ! git diff --quiet origin/HEAD HEAD -- deps/build.jl; then unset JULIA_TZ_VERSION; fi
```

Comment:
I could remove the entire `script` section if I would move this part into a `before_script` section. I'll note that the unsetting of the environmental variable does seem to persist between these stages but I'm reluctant to trust that this will continue to be the case.